logging_http_client/logging_http_client_config.py

```diff
@@ -116,3 +116,22 @@ def enable_response_body_logging(enable: bool = True) -> None:
         these are for modifying the default logging setup)
     """
     config.set_response_body_logging_enabled(enable)
+
+
+def set_logging_level(level: str | int = 20) -> None:
```

Comment:
**suggestion(non-blocking):** This is in an interesting dynamic-typing implementation for the log level.

If you notice, you had to supply (in comments) the enumeration of values for the level and integer. This is a sign that this argument is better modeled as an enum.

Would you like to apply this refactor? i.e. create an enum of: 'DEBUG', 'INFO', 'WARNING', 'ERROR', or 'CRITICAL'.

Something like so:

```python
import logging
from enum import Enum

class LogLevel(Enum):
    DEBUG = logging.DEBUG
    INFO = logging.INFO
    WARNING = logging.WARNING
    ERROR = logging.ERROR
    CRITICAL = logging.CRITICAL

def set_logging_level(level: LogLevel = LogLevel.INFO) -> None:
    """
    Set the logging level for the logger. Default is LogLevel.INFO.

    The level should be an instance of LogLevel Enum, such as LogLevel.DEBUG, LogLevel.INFO, etc.
    """
    numeric_level = level.value
    config.set_logging_level(numeric_level)
```

This is optional :)

Reply:
Hey Thanks for the suggestion, I like your implementation more, I will implement this, Just one question, how do I set the default value for the `_loggin_level` global variable in the `logging_http_client_config_globals.py`. Should I create the enum in that file itself? Or make it so that we implicitly set a logging level at the start if not set already?

Reply:
> Should I create the enum in that file itself? Or make it so that we implicitly set a logging level at the start if not set already?

Let's create the enum in a file, in which you can then import in the config and use the INFO as the default value.